target/linux/ath79/patches-6.1/312-dt-bindings-PCI-qcom-ar7240-adds-binding-doc.patch

```diff
@@ -39,7 +39,7 @@ Signed-off-by: John Crispin <john@phrozen.org>
 +- interrupt-controller: define to enable the builtin IRQ cascade.
 +
 +* Example for qca9557
-+	pcie-controller@180c0000 {
++	pcie@180c0000 {
 +		compatible = "qcom,ar7240-pci";
 +		#address-cells = <3>;
 +		#size-cells = <2>;
```

Comment:
This is something unrelated:
The ath79 PCIe driver was broken on kernel 6.1. Now I'm using a hack to workaround it, see https://github.com/openwrt/openwrt/commit/54758cf24bd8123c724c59805a21d940db0b058c. Perhaps someone in this thread has enough knowledge to fix it by reorganizing the DTS PCIe node.

Reply:
Unfortunately I don’t have such knowledge.

Reply:
@neheb can we move this to another pr and drop the commit from here?

Reply:
Hmm? This is controversial?

Reply:
hm? no? From what I'm reading this cause regression and I think we should investigate it further. Splitting it from this pr permits the other commits to be merged

Reply:
If you mean @DragonBluep ’s commit, that’s not related to this change. He’s commenting that the PCIe driver and dts need further fixing because of upstream changes.

Reply:
Oh then I was totally confused by this. Sorry!